app/assets/stylesheets/base/banners.scss

```diff
@@ -9,9 +9,13 @@ Banners - Notification messages from the application displaying info to user
 
 // Generic styling for a banner
 .banner {
```

Comment:
Not sure if I'm a huge fan of the floating banner, to be honest, but that's just a stylistic choice I suppose

Reply:
I changed it because I was trying to implement the call to add/remove the team via JavaScript Asynchronously. The idea being if the user was scrolled down the page they'd still see the banner. However, I couldn't get the Async call in JS to work exactly as I wanted it too, so I ditched that and just reverted to forms. I had already changed the banners so I wasn't sure if I wanted to revert it without at least showing it to someone haha. But we can definitely revert it.